spec/unit/provider/oneview_logical_interconnect_spec.rb

```diff
@@ -198,3 +196,65 @@
     end
   end
 end
+
+api_versions = 2000
+resource_types = OneviewSDK.resource_named(:LogicalInterconnect, api_versions, :C7000)
+
+describe provider_class, unit: true do
+  include_context 'shared context Oneview API 2000'
+
+  let(:resource) do
+    Puppet::Type.type(:oneview_logical_interconnect).new(
+      name: 'LI',
+      ensure: 'present',
+      data:
+          {
+            'name' => 'Encl2-my enclosure logical interconnect group',
+            'internalNetworks' => ['NET'],
```

Comment:
is it required to provide all this data here..?

Reply:
i tried adding few data here, its failed since exist? will raise confilcts.